src/api/app/controllers/source_attribute_controller.rb

```diff
@@ -43,8 +43,8 @@ def delete
     unless attrib
       raise ActiveRecord::RecordNotFound, "Attribute #{params[:attribute]} does not exist"
     end
-    unless User.current.can_create_attribute_in?(@attribute_container, @at)
-      raise ChangeAttributeNoPermission, "User #{user.login} has no permission to change attribute"
+    unless User.possibly_nobody.can_create_attribute_in?(@attribute_container, @at)
```

Comment:
@coolo 

why ```possibly_nobody``` and not ```session!``` ?

Reply:
basically the same as above - so for consistency I better change to session! too